### PR TITLE
FilVideo project. Join Slingshot

### PR DESCRIPTION
**Pull request by jackoelv**

### CODE REVIEW

File: participants/FilVideo.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# FilVideo
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
```
Comment (cwaring) [inline participants/FilVideo.md:5]:
```suggestion
Before submitting a PR containing your team's project information:
- Make sure you rename this file to contain your project's name
- Respond to add your response in each section below
```
please remove the default info.

**Review by cwaring — CHANGES_REQUESTED** [review]
Hi @jackoelv, thanks for your submission. Please amend the minor suggestion and restore the original template.md and we can accept this application. Thanks

**Comment by jackoelv** [comment]
default info removed. thanks.

**Comment by cwaring** [comment]
thanks @jackoelv you also need to restore the original `template.md` file, otherwise I'm unable to merge this PR

File: participants/FilVideo.md (hunk 1 above)
Comment (jackoelv) [reply participants/FilVideo.md]: add template.md to project.

**Review by cwaring — APPROVED** [review]

**Comment by jackoelv** [comment]
@cwaring  need one more approval.

**Comment by cwaring** [comment]
@dineshshenoy mind queuing this up as reviewer two?

**Review by GregTheGreek — APPROVED** [review]

**Pull request merged**